index.html

```diff
@@ -7,7 +7,10 @@
 <body>
   <div class="center reamde">
     <h1> Spring 2015!</h1>
-    <a href="david.html">David Zabner's page</a>
+    <ul>
+    	<li><a href="david.html">David Zabner's page</a></li>
```

Comment:
I noticed the bullets but figured they were keeping in line with the ugly, er, utilitarian theme that you already had going. Happy to make the index page prettier.